appveyor.yml

```diff
@@ -9,6 +11,7 @@ version: '$(major).$(minor).$(release).{build}'
 build_script:
   - cmd: msbuild OSPSuite.SimModel.sln /property:Configuration=Debug;Platform=Win32 /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
   - cmd: msbuild OSPSuite.SimModel.sln /property:Configuration=Release;Platform=Win32 /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+  - cmd: msbuild OSPSuite.SimModel.sln /property:Configuration=ReleaseX64;Platform=x64 /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
Why use ReleaseX64 and not simply release as normal? The difference between 32 and 64 is already in the platform

Reply:
And the expected folder structure is release/32 and release/64 no?

Reply:
>And the expected folder structure is release/32 and release/64 no?

If you are talking about nuget package: it's under **bin\native\x64** and **bin\native\x86**.

>Why use ReleaseX64 and not simply release as normal? The difference between 32 and 64 is already in the platform

Guess you are right, should be done this way. Will check if it's not too much effort to change this